driver-embedded/src/main/com/mongodb/embedded/client/MongoEmbeddedSettings.java

```diff
@@ -77,6 +126,40 @@ public Builder libraryPath(final String libraryPath) {
             return this;
         }
 
+        /**
+         * Sets the yaml configuration for mongod.
+         *
+         * @param yamlConfig the yaml configuration for mongod
+         * @return this
+         * @mongodb.driver.manual reference/configuration-options/
+         */
+        public Builder yamlConfig(final String yamlConfig) {
+            this.yamlConfig = yamlConfig;
+            return this;
+        }
+
+        /**
+         * Sets the logging level for the mongod.
+         *
+         * @param logLevel the library path for mongod
+         * @return this
+         */
+        public Builder logLevel(final LogLevel logLevel) {
+            this.logLevel = logLevel;
+            return this;
+        }
+
+        /**
+         * Sets any application data to included in the logging
+         *
+         * @param applicationData any application data to be included in the logging
+         * @return this
+         */
+        public Builder applicationData(final String applicationData) {
```

Comment:
`userData` didn't make much sense in the context of the driver, so I changed the name to `applicationData`, to try and clarify its purpose.  I also fixed the type to String, because its only ever used by the `LogCallback`.